python/ray/dag/compiled_dag_node.py

```diff
@@ -539,6 +540,10 @@ def __init__(
                 executions is beyond the DAG capacity, the new execution would
                 be blocked in the first place; therefore, this limit is only
                 enforced when it is smaller than the DAG capacity.
+            max_in_flight_requests: The maximum number of in-flight requests that
+                are allowed to be buffered. Before submitting more requests,
+                the caller is responsible for calling ray.get to clear finished
+                in-flight requests.
```

Comment:
Add: otherwise `RayChannelTimeoutError` may be thrown.

Reply:
We should mention the side effect: i.e., memory overhead if a large number is used. Users may blindly use a large value.